README.md

```diff
@@ -32,9 +36,9 @@
   Use this Configuration Page to set the collector version, script version, and specify
   the uri of the tracking script.
 
-## Getting Help
-
-For any questions regarding this project, please contact the GDX Analytics Team.
+## Getting Help or Reporting an Issue
+ 
+For any questions regarding this project, or for inquiries about starting a new analytics account, please contact the GDX Analytics Team.
 
 ## Contributors
 
```

Comment:
needs to change to 'The GDX Analytics Team are the main contributors to this project and maintain the code.'

Reply:
We arent changing the contributing instructions yet.

Reply:
I saw Dan's comment in the chat, so I updated the wording.